src/GeneralStateTestsFiller/stEWASMTests/createCodeWithRevertFiller.yml

```diff
@@ -0,0 +1,155 @@
+
+            (call $revert
+              (i32.const 512) ;; dataOffset
+              (i32.const 2)   ;; dataLength
+            )
+          )
+        )
+    
+    deadbeef00000000000000000000000000000000:
+      balance: '100000000000'
+      nonce: ''
+      storage: {}
+      code: |
+        (module
+          (import "ethereum" "create" (func $create (param i32 i32 i32 i32) (result i32)))
+          (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+          (import "ethereum" "getExternalCodeSize" (func $getExternalCodeSize (param i32) (result i32)))
+          (import "ethereum" "externalCodeCopy" (func $externalCodeCopy (param i32 i32 i32 i32)))
+          (memory 1)
+
+          ;; copy the code from this account
+          (data (i32.const 192) "\a0\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\01")
+
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+
+          (func $main
+            ;; locals
+            (local $codeMemLoc i32)
+            (local $codeLen i32)
+            (local $address i32)
+            (local $memPointerStorageKey1 i32)
+            (local $memPointerStorageKey2 i32)
+            (local $memPointerCreateValue i32)
+            (local $memPointerCreateResult i32)
+            (local $memPointerCreateRetCode i32)
+
+            ;; Location and length of code stored in (data) section above
+            (set_local $codeMemLoc (i32.const 256))
+            (set_local $codeLen (i32.const 160))
+            (set_local $address (i32.const 192))
+
+            ;; memory layout and pointers
+            (set_local $memPointerStorageKey1 (i32.const 0))
+            (set_local $memPointerStorageKey2 (i32.const 32))
+            (set_local $memPointerCreateValue (i32.const 64))
+            (set_local $memPointerCreateResult (i32.const 96))
+            (set_local $memPointerCreateRetCode (i32.const 128))
+
+            ;; initialize
+            (i32.store (get_local $memPointerStorageKey1) (i32.const 0))
+            (i32.store (get_local $memPointerStorageKey2) (i32.const 1))
+            ;; call create with value transfer 0
+            (i32.store (get_local $memPointerCreateValue) (i32.const 0))
+
+            ;; get external code
+            (set_local $codeLen
+              (call $getExternalCodeSize (get_local $address)))
+
+            (call $externalCodeCopy
+              (get_local $address)
+              (get_local $codeMemLoc)
+              (i32.const 0)
+              (get_local $codeLen))
+
+            (i32.store (get_local $memPointerCreateRetCode)
+              (call $create
+                ;; value offset
+                (get_local $memPointerCreateValue)
+                ;; data offset
+                (get_local $codeMemLoc)
+                ;; data length
+                (get_local $codeLen)
+                ;; result offset (new contract address)
+                (get_local $memPointerCreateResult)))
```

Comment:
I think it is a bad idea for the test case to overwrite the value pointer with the resulting address:
```
            (i32.store (get_local $memPointerStorageKey1) (i32.const 0))
             (i32.store (get_local $memPointerStorageKey2) (i32.const 1))
             ;; call create with value transfer 0
             (i32.store (get_local $memPointerCreateValue) (i32.const 0))
```